src/pp_ast.mli

```diff
@@ -60,28 +60,46 @@ module Config : sig
           be. *)
 end
 
+type simple_val =
+  | Unit
+  | Int of int
+  | String of string
+  | Bool of bool
+  | Char of char
+  | Array of simple_val list
+  | Float of float
+  | Int32 of int32
+  | Int64 of int64
+  | Nativeint of nativeint
+  | Record of (string * simple_val) list
+  | Constr of string * simple_val list
+  | Tuple of simple_val list
+  | List of simple_val list
+  | Special of string
+
+type printer = Format.formatter -> simple_val -> unit
```

Comment:
I guess we could remove this alias in favor of a simpler `repr pp` don't you think?